bloom-instance/logging.tf

```diff
@@ -1,3 +1,24 @@
 resource "aws_s3_bucket" "logging_bucket" {
   bucket_prefix = "${var.name_prefix}-logging"
 }
+
+resource "aws_s3_bucket_policy" "log_bucket_policy" {
+  bucket = aws_s3_bucket.logging_bucket.id
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Sid = "AllowWritesToLogBucket"
+        Action = [
+          "s3:PutObject",
+        ]
+        Effect = "Allow"
+        Principal = {
+          AWS = local.elb_service_account_arn
```

Comment:
should we disambiguate between ALBs and ELBs?

Reply:
Technically speaking, this var references the ARN for the Elastic Load Balancer service's AWS account and isn't specific to ALBs.  If we had to use the same ARN for an NLB later on, I think it might be more confusing if it were named after ALBs